app/assets/stylesheets/elements/_alert.scss

```diff
@@ -1,28 +1,46 @@
 @keyframes move-in-alert {
   from {
-    transform: translateY(200%);
+    transform: translateY(100vh);
   }
 
   to {
     transform: translateY(0);
   }
 }
 
-.alert {
+.alerts {
   position: fixed;
   right: $margin / 3;
   bottom: $margin / 3;
+  display: flex;
+  flex-direction: column;
+  align-items: end;
+  gap: $margin / 3;
+  width: min(35em, 75%);
+  z-index: 2000;
+}
+
+.alerts__alert {
   padding: .5em 1em;
+  display: flex;
+  align-items: center;
+  gap: 1em;
```

Comment:
Didn't know they renamed `grid-gap` to `gap`, neat!

Reply:
Yep, `grid-gap` is technically deprecated, since the `gap` property applies to [flex, grid, and any multi-column layout](https://css-tricks.com/almanac/properties/g/gap/).